src/interpreter/ArithmeticHelper.ts

```diff
@@ -47,7 +47,7 @@ export class ArithmeticHelper {
   private readonly actualEps: number
   constructor(
     private readonly config: Config,
-    private readonly dateTimeHelper: DateTimeHelper,
+    public readonly dateTimeHelper: DateTimeHelper,
     private readonly numberLiteralsHelper: NumberLiteralHelper,
```

Comment:
Why not make both public? If this is the recommended way to use them, they should be available